orchagent/portsorch.cpp

```diff
@@ -738,6 +749,21 @@ bool PortsOrch::removeSubPort(const string &alias)
     m_portList[parentPort.m_alias] = parentPort;
 
     m_portList.erase(it);
+
+    // Restore hostif vlan tag for the parent port when the last subport is removed
+    if (parentPort.m_child_ports.empty())
+    {
+        if (!parentPort.m_bridge_port_id)
+        {
+            if (!setHostIntfsStripTag(parentPort, SAI_HOSTIF_VLAN_TAG_STRIP))
+            {
+                SWSS_LOG_WARN("Failed to set %s for hostif of port %s",
```

Comment:
What is the thought for `WARN` instead of `ERROR` as above?

Reply:
Copy & paste from the log right above in the function. Didn't notice that there is no return false.